src/Pagination/model.js

```diff
@@ -0,0 +1,166 @@
+export function buildPaginationModel(pageCount, currentPage, showPages, marginPageCount, surroundingPageCount) {
+  const pages = []
+
+  if (showPages) {
+    const pageNums = new Set()
```

Comment:
Do you automatically include a Set polyfill? Or are you in the enviable spot of not supporting IE?

Reply:
Dammit IE

Reply:
Depends on your support requirements; IE11 supports SOME of it but not much. I wouldn't recommend it without a polyfill though.